lib/config/definitions.js

```diff
@@ -911,6 +911,13 @@ const options = [
     type: 'integer',
     default: 0, // no limit
   },
+  {
+    name: 'bbUseDefaultReviewers',
+    releaseStatus: 'unpublished',
```

Comment:
Remove this line

Reply:
you mean `releaseStatus` ? Ok, then i have to add the docu for that, otherwise ta test fails